index.bs

```diff
@@ -378,6 +410,8 @@ An attribution trigger is a [=struct=] with the following items:
 :: An [=attribution filter map=].
 : <dfn>debug key</dfn>
 :: Null or a non-negative 64-bit integer.
+: aggregatable trigger
```

Comment:
```suggestion
: <dfn>aggregatable trigger</dfn>
```

Reply:
Will add this when adding histogram creation algorithm.